File: app/src/main/java/com/xmartlabs/template/module/AndroidModule.java
package com.xmartlabs.template.module;

import android.app.Application;

import com.xmartlabs.base.core.module.CoreAndroidModule;
import com.xmartlabs.template.model.BuildInfo;

public class AndroidModule extends CoreAndroidModule {
  public AndroidModule(Application application) {
    super(application);
  }

  @Override
  public com.xmartlabs.base.core.model.BuildInfo provideBuildInformation() {
    return new BuildInfo();
  }
}

File: app/src/main/java/com/xmartlabs/template/module/AndroidModule.kt
package com.xmartlabs.template.module

import android.app.Application

import com.xmartlabs.bigbang.core.module.CoreAndroidModule
import com.xmartlabs.template.model.BuildInfo
import com.xmartlabs.bigbang.core.model.BuildInfo as CoreBuildInfo

class AndroidModule(application: Application) : CoreAndroidModule(application) {
  override fun provideBuildInformation(): CoreBuildInfo = BuildInfo()
}
